src/flash.sol

```diff
@@ -54,16 +62,21 @@ contract DssFlash is ReentrancyGuard {
     }
 
     // --- Mint ---
-    function mint(address _receiver, uint256 _amount, bytes calldata _params) external {
+    function mint(
+        address _receiver,      // address of conformant IFlashMintReceiver
+        uint256 _amount,        // amount to flash mint [rad]
+        bytes calldata _data    // calldata
+    ) external lock {
         require(_amount > 0, "DssFlash/amount-zero");
         require(_amount <= line, "DssFlash/ceiling-exceeded");
+        require(_data.length > 0, "DssFlash/empty-calldata");
```

Comment:
I'm curious as to why this requirement was added. It shouldn't be a problem to send empty calldata.

Reply:
I just saw this comment and was curious myself. This require shouldn't be here. An empty `bytes` will have a length of `0`, and isn't strictly necessary in a flash loan implementation.

![Screenshot from 2020-09-21 16-07-07](https://user-images.githubusercontent.com/2374718/93816084-02409d80-fc25-11ea-931c-ca820e5ace88.png)

Reply:
Yeah this arbitrary data is just for code re-use on the receiver.